### PR TITLE
Add staged VibeVoice text-to-speech support

### PR DESCRIPTION
**Pull request by justinchuby**
## Summary

- add Transformers-native `vibevoice` support pinned to `vibevoice/VibeVoice-1.5B-hf@edc39f80f5cae656da37baf8faa8f5502bf7081f`; record `microsoft/VibeVoice-1.5B@c00898d257e6b46004e3e2866a47534085fb685a` as the provenance root, not the executable source
- export eight explicit ONNX stages: reference audio encoder, acoustic projection, prompt embedding, mask-safe Qwen2 decoder, diffusion head, streaming acoustic decoder, streaming semantic encoder, and semantic projection
- preserve 28-layer KV state and 34-slot acoustic/semantic convolution state; keep standard Attention for the CFG negative-cache suffix reset because GQA's prefix-length ABI cannot represent that mask
- package pinned tokenizer/chat template/processor/generation assets and graph-derived advisory metadata
- add provenance-recorded 24 kHz LibriSpeech fixture plus text-only, reference-audio, and two-speaker contracts

## Evidence

- L1: `tests/build_graph_test.py -k vibevoice` — 8 passed
- weight alignment: `tests/weight_alignment_test.py -k vibevoice` — 2 passed; all 1,204 native checkpoint keys route to trained stage parameters
- L2: `tests/arch_validation_test.py -k vibevoice` — 3 passed against the full pinned 1.5B config
- L3 synthetic: `tests/synthetic_parity_test.py -k vibevoice` — 1 passed; every stage matches native HF, including non-identity latent scale/bias and ONNX decoder→semantic feedback
- L3 real weights / L4 / L5: `tests/vibevoice_golden_test.py -m "golden or integration"` on CUDA — 8 passed
  - exact generated control tokens: `[151654, 151654, 151654]`
  - exact continuous-latent shape: `[1, 3, 64]`
  - exact waveform shape: `[1, 1, 9600]` (3 × 3,200 samples)
  - semantic checks: finite, >99.9% nonzero, non-degenerate RMS/peak, >100 zero crossings, native waveform summaries/first samples
  - real stage-by-stage HF parity and CPU/CUDA generation parity included
- processor: pinned text-only `(1,38)`, reference `(1,116)` with 69 audio placeholders, and two-speaker `(1,203)` with 138 ordered placeholders
- YAML/schema: 276 passed
- combined VibeVoice gates: 19 passed
- CLI: weighted FP16 CUDA-targeted package saved all eight stages plus `audio_processor.json`, `processor_config.json`, `generation_config.json`, tokenizer, chat template, revision metadata, and advisory runtime contract
- Olive: real weighted acoustic projector quantized with RTN INT4 to `MatMulNBits`; CPU/CUDA max difference `0.001953125`, correlation to FP16 `0.9999083`, finite non-degenerate output
- lint: exact `lintrunner -a` — clean; specialist review rerun after fixes — no findings

## Waivers

- BF16 CUDA end-to-end generation: ORT 1.26 on the RTX A1000 cannot assign the codec's BF16 `ConvTranspose` nodes. BF16 reference-audio encoder parity was still measured (`corr=0.9999813`), while full CUDA L4/L5 and CPU/CUDA parity use FP16. No silent dtype downgrade is performed.
- `onnx-genai` does not yet orchestrate VibeVoice's dual decoder caches, DPM-Solver loop, and streaming convolution state. The package emits exact graph/processor contracts and an explicit `unsupported-by-tested-runtime` advisory rather than claiming downstream support.
- `lintrunner f --output oneline --all-files` applied formatting but the Ruff subprocess hit Windows `WinError 206` from the repository-wide path list in this deep worktree. The required exact changed-file `lintrunner -a` completed cleanly after pinned linter initialization.
- the full non-integration repository run recorded 9,810 passes; remaining failures were unrelated pre-existing Windows symlink-privilege and unrelated numerical/hash cases. Every changed VibeVoice/CLI/optimizer surface passes its targeted suite.

### CODE REVIEW

File: tests/e2e_golden_test.py [hunk 1]
```diff
@@ -2197,6 +2197,11 @@ class TestL4CheckpointVerified:
 
     @pytest.mark.parametrize("case", _L4_CASES)
     def test_prefill_argmax_matches_golden(self, case: GoldenTestCase) -> None:
+        if case.task_type == "text-to-speech":
+            pytest.skip(
+                "Continuous-token TTS uses tests/vibevoice_golden_test.py "
+                "(control logits, latent frames, and waveform semantics)."
+            )
```
[the remark this answers is not in the document]
Comment (justinchuby) [reply tests/e2e_golden_test.py:2204]: Fixed on the rebased head. The GPU L4 and L5 reusable workflows now invoke the dedicated VibeVoice runner for all-model, affected-model, and exact-case dispatches. L5 is explicitly marked `generation`; L4 excludes generation tests so each gate exercises only its intended tier. Exact VibeVoice case selection is validated during collection.

File: tests/synthetic_parity_test.py [hunk 2]
```diff
@@ -46,6 +46,13 @@
 logger = logging.getLogger(__name__)
 
 
+def test_vibevoice_synthetic_pipeline_parity():
+    """Run the dedicated eight-stage continuous-token parity harness."""
+    from mobius.models.vibevoice_test import test_vibevoice_synthetic_stage_parity
+
+    test_vibevoice_synthetic_stage_parity()
```
[the remark this answers is not in the document]
Comment (justinchuby) [reply tests/synthetic_parity_test.py]: Fixed in `64f0d982`. The implementation is now `run_vibevoice_synthetic_stage_parity()` (not a pytest test), and `tests/synthetic_parity_test.py` is the sole L3 entry point. Collection confirms exactly one synthetic pipeline parity case, while the independent model-contract tests remain separately collected.
